recipes/default.rb

```diff
@@ -28,6 +28,16 @@
   end
 end
 
+# Create the mongodb user if not exist
+user node['mongodb3']['user'] do
```

Comment:
@barata 
The mongod and agent package installs the user and group. but mongos package does not. I'm gonna leave mongodb package to install the user/group for mongod and agent.